docs/.vuepress/config.js

```diff
@@ -136,6 +136,10 @@ module.exports = {
                       ["/ReverseSSHtunnel", "Reverse SSH Tunnel"]
                     ]
                   }
```

Comment:
The indentation is awry (we can fix it later on), but this line is missing a trailing comma, which should lead to a build error.

Reply:
It just did yeah.

Was also fixing the spacing problem that I still have.

Reply:
Still failing check.